src/toolbars/examples/toolbar.js

```diff
@@ -104,7 +113,16 @@
         </pf-list-view>
       </div>
       <div class="col-md-12" ng-if="viewType == 'cardView' && showComponent">
-        <pf-card-view config="listConfig"
+        <div ng-if="items.length === 0">
+          <div class="pf-empty-state">
+            <div ng-if="filterConfig.appliedFilters.length > 0">
+              <pf-empty-state config="noItemsConfig"></pf-empty-state>
+            </div>
+          </div>
+        </div>
```

Comment:
Rather than repeating/adding `<pf-empty-state/>`'s to the demo HTML, since we are already passing in `empty-state-config`'s to each of the List,Card,Table Views, you can just dynamically set it:

```
<pf-list-view config="listConfig"
                      page-config="pageConfig"
                      items="items"
                      empty-state-config="items.length === 0 ? noItemsConfig : emptyStateConfig">
```
Same for the Card and Table views.   Since this will use the `<pf-empty-state/>' embedded in each of the [views](https://github.com/patternfly/angular-patternfly/blob/master/src/views/listview/list-view.html#L75) this will also fix the TableView's 'no results' empty state container being wider than the toolbar.

Reply:
good idea

Reply:
Completed, thanks!